app/src/pages/evaluators/EvaluatorsTable.tsx

```diff
@@ -0,0 +1,360 @@
+import {
+  startTransition,
+  useCallback,
+  useEffect,
+  useMemo,
+  useRef,
+} from "react";
+import { graphql, usePaginationFragment } from "react-relay";
+import { useNavigate } from "react-router";
+import {
+  ColumnDef,
+  flexRender,
+  getCoreRowModel,
+  getSortedRowModel,
+  SortingState,
+  Updater,
+  useReactTable,
+} from "@tanstack/react-table";
+import { css } from "@emotion/react";
+
+import { Flex, Icon, Icons, Text, Token, View } from "@phoenix/components";
+import { TextCell } from "@phoenix/components/table";
+import { IndeterminateCheckboxCell } from "@phoenix/components/table/IndeterminateCheckboxCell";
+import { selectableTableCSS } from "@phoenix/components/table/styles";
+import { TimestampCell } from "@phoenix/components/table/TimestampCell";
+import { EvaluatorsTable_evaluators$key } from "@phoenix/pages/evaluators/__generated__/EvaluatorsTable_evaluators.graphql";
+import {
+  EvaluatorSort,
+  EvaluatorsTableEvaluatorsQuery,
+} from "@phoenix/pages/evaluators/__generated__/EvaluatorsTableEvaluatorsQuery.graphql";
+import { useEvaluatorsFilterContext } from "@phoenix/pages/evaluators/EvaluatorsFilterProvider";
+
+const PAGE_SIZE = 100;
+
+const convertEvaluatorSortToTanstackSort = (
+  sort: EvaluatorSort | null | undefined
+): SortingState => {
+  if (!sort) return [];
+  return [{ id: sort.col, desc: sort.dir === "desc" }];
+};
+
+const convertTanstackSortToEvaluatorSort = (
+  sorting: SortingState
+): EvaluatorSort | null | undefined => {
+  if (sorting.length === 0) return null;
+  const col = sorting[0].id;
+  if (
+    col !== "name" &&
+    col !== "kind" &&
+    col !== "createdAt" &&
+    col !== "updatedAt"
+  ) {
```

Comment:
```suggestion
  if (
    col in SORTABLE_COLUMNS
```

I would refactor to make it self documenting.